composer/lib/dependabot/composer/helpers.rb

```diff
@@ -16,13 +16,13 @@ module Helpers
       def self.composer_version(composer_json, parsed_lockfile = nil)
         if parsed_lockfile && parsed_lockfile["plugin-api-version"]
           version = Composer::Version.new(parsed_lockfile["plugin-api-version"])
```

Comment:
We could directly return the version here, then let callers decide their desired granularity... ie, metrics probably `2.4`, `2.5`, etc.

But right now we don't need that, and I expect by the time we do we'll have dropped support for composer v1 (https://github.com/dependabot/dependabot-core/issues/6298), so the implementation will be a bit simpler.

So will leave that for future PR.

I did add the necessary unit tests so that if/when that happens, they'll flag the callers that need changing.